MapboxNavigation/Resources/yo.lproj/Localizable.strings

```diff
@@ -0,0 +1,126 @@
+/* Delimiter between lines in an address when displayed inline */
+"ADDRESS_LINE_SEPARATOR" = ", ";
+
+/* Title on arrival action sheet */
+"CARPLAY_ARRIVED" = "O ti de";
+
+/* Message on arrival action sheet */
+"CARPLAY_ARRIVED_MESSAGE" = "Kí l'ó wù ẹ láti ṣe?";
+
+/* Name of the waypoint associated with the current location */
+"CARPLAY_CURRENT_LOCATION" = "Ipo ti isiyi";
+
+/* Title for dismiss button */
+"CARPLAY_DISMISS" = "Fi silẹ";
+
+/* Title for end navigation button */
+"CARPLAY_END" = "Ipari";
+
+/* Title on the exit button in the arrival form */
+"CARPLAY_EXIT_NAVIGATION" = "Lilọ kiri kuro";
+
+/* Title for feedback template in CarPlay */
+"CARPLAY_FEEDBACK" = "Idahun";
+
+/* Title for start button in CPTripPreviewTextConfiguration */
+"CARPLAY_GO" = "Lọ";
+
+/* Title for alternative routes in CPTripPreviewTextConfiguration */
+"CARPLAY_MORE_ROUTES" = "Awọn ipa-ọna diẹ sii";
+
+/* Title for mute button */
+"CARPLAY_MUTE" = "Mute";
+
+/* Title for overview button in CPTripPreviewTextConfiguration */
+"CARPLAY_OVERVIEW" = "Akopọ";
+
+/* Title for rating template in CarPlay */
+"CARPLAY_RATE_RIDE" = "Ṣe ayẹwo gigun rẹ";
+
+/* Title on rate button in CarPlay */
+"CARPLAY_RATE_TRIP" = "Ṣe atọkasi irin-ajo rẹ";
+
+/* Message when search returned zero results in CarPlay */
+"CARPLAY_SEARCH_NO_RESULTS" = "Ko si awọn esi";
+
+/* Alert title that shows when feedback has been submitted */
+"CARPLAY_SUBMITTED_FEEDBACK" = "Ti firanṣẹ";
+
+/* Title for unmute button */
+"CARPLAY_UNMUTE" = "Ṣi i";
+
+/* Format string for indicating the device volume is low; 1 = device model */
+"DEVICE_VOLUME_LOW" = "%@Didun kekere";
```

Comment:
The SDK replaces `%@` with the device model. Is it correct to say “iPhoneDidun kekere” or “iPadDidun kekere”?